core/src/main/scala/akka/persistence/jdbc/journal/dao/AkkaSerializerJournalDao.scala

```diff
@@ -0,0 +1,121 @@
+package akka.persistence.jdbc.journal.dao
+
+import akka.NotUsed
+import akka.persistence.jdbc.config.{ BaseDaoConfig, JournalConfig }
+import akka.persistence.jdbc.journal.dao.AkkaSerialization.AkkaSerialized
+import akka.persistence.jdbc.journal.dao.JournalTables.JournalAkkaSerializationRow
+import akka.persistence.journal.Tagged
+import akka.persistence.{ AtomicWrite, PersistentRepr }
+import akka.serialization.{ Serialization, Serializers }
+import akka.stream.Materializer
+import akka.stream.scaladsl.Source
+import slick.jdbc.JdbcBackend.Database
+import slick.jdbc.JdbcProfile
+
+import scala.collection.immutable
+import scala.collection.immutable.{ Nil, Seq }
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.Try
+
+/**
+ * A [[JournalDao]] that uses Akka serialization to serialize the payload and store
+ * the manifest and serializer id used.
+ */
+class AkkaSerializerJournalDao(
+    val db: Database,
+    val profile: JdbcProfile,
+    val journalConfig: JournalConfig,
+    serialization: Serialization)(implicit val ec: ExecutionContext, val mat: Materializer)
+    extends JournalDao
+    with BaseDao[(JournalAkkaSerializationRow, Set[String])]
+    with BaseJournalDaoWithReadMessages {
+
+  import profile.api._
+
+  override def baseDaoConfig: BaseDaoConfig = journalConfig.daoConfig
+
+  override def writeJournalRows(xs: immutable.Seq[(JournalAkkaSerializationRow, Set[String])]): Future[Unit] = {
+    db.run(queries.writeJournalRows(xs).transactionally).map(_ => ())
+  }
+
+  val queries = new JournalQueries(profile, journalConfig.journalTableConfiguration)
+
+  queries.JournalTable.schema.createIfNotExistsStatements.foreach(println)
+  queries.TagTable.schema.createIfNotExistsStatements.foreach(println)
+
+  override def delete(persistenceId: String, maxSequenceNr: Long): Future[Unit] = {
+    val actions: DBIOAction[Unit, NoStream, Effect.Write with Effect.Read] = for {
+      _ <- queries.markJournalMessagesAsDeleted(persistenceId, maxSequenceNr)
+      highestMarkedSequenceNr <- highestMarkedSequenceNr(persistenceId)
+      _ <- queries.delete(persistenceId, highestMarkedSequenceNr.getOrElse(0L) - 1)
```

Comment:
Yep let's do that under a diff PR and have a deleted_to table if we think it is important enough.

It would be easy to make the change in a backward compatible way so no immediate need